Dockerfile

```diff
@@ -9,5 +9,5 @@ ADD scm-source.json /scm-source.json
 EXPOSE 8080
 
 # run the server when a container based on this image is being run
-ENTRYPOINT java -jar -Djava.security.egd=file:/dev/./urandom nakadi.jar
+ENTRYPOINT java $JAVA_OPTS -jar -Djava.security.egd=file:/dev/./urandom nakadi.jar
```

Comment:
But still there's no guarantee that it will supported in other JVMs. I would feel more safe with passing it directly when running java. This is an approach a lot of java apps use.